sample/src/main/java/com/airbnb/deeplinkdispatch/sample/MainActivity.java

```diff
@@ -28,7 +28,13 @@
 import androidx.core.app.ActivityCompat;
 import androidx.core.app.TaskStackBuilder;
 
-@DeepLink({ "dld://classDeepLink", "http://example.com/foo{arg_end}", "http://example.com/{arg_start}bar", "dld://example.com/deepLink" })
+@DeepLink({
+  "dld://classDeepLink",
+  "http://example.com/foo{arg_end}",
+  "http://example.com/{arg_start}bar",
+  "dld://example.com/deepLink",
+  "https://www.example.com/-replaceable-path-variable-/bar"
```

Comment:
Didn't we say we use something like `###somevalue###` for the placeholder? `-replaceable-path-variable-` is still to easy to be matched by something else.